core/comment.js

```diff
@@ -10,43 +10,42 @@
  */
 'use strict';
 
-goog.provide('Blockly.Comment');
-
-goog.require('Blockly.browserEvents');
-goog.require('Blockly.Bubble');
-goog.require('Blockly.Css');
-goog.require('Blockly.Events');
+goog.module('Blockly.Comment');
+goog.module.declareLegacyNamespace();
+
+const Block = goog.requireType('Blockly.Block');
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const Bubble = goog.require('Blockly.Bubble');
+const Coordinate = goog.requireType('Blockly.utils.Coordinate');
+const Events = goog.require('Blockly.Events');
+const Icon = goog.require('Blockly.Icon');
+const Size = goog.requireType('Blockly.utils.Size');
+const Svg = goog.require('Blockly.utils.Svg');
+const WorkspaceSvg = goog.requireType('Blockly.WorkspaceSvg');
+const {CommentModel} = goog.requireType('Blockly.Block');
```

Comment:
This shouldn't be required twice, instead you should use `Block.CommentModel` in the JsDoc reference.

Reply:
Done